packages/rich-text-editor/src/_mixins.scss

```diff
@@ -0,0 +1,41 @@
+@import "@kaizen/design-tokens/sass/spacing";
+@import "@kaizen/design-tokens/sass/typography";
+
+@mixin contentStyles {
+  font-family: $typography-paragraph-body-font-family;
+  font-weight: $typography-paragraph-body-font-weight;
+  font-size: $typography-paragraph-body-font-size;
+  line-height: $typography-paragraph-body-line-height;
+  letter-spacing: $typography-paragraph-body-letter-spacing;
+  position: relative;
+  white-space: pre-wrap;
+
+  > p {
+    margin: 0 0 $spacing-sm;
+  }
+
+  > p:last-child {
+    margin-bottom: 0;
+  }
+
+  ol {
+    list-style-type: decimal;
+    ol {
+      list-style-type: lower-alpha;
+      ol {
+        list-style-type: lower-roman;
+        @content;
```

Comment:
One thing I just wanted to call out here is if they go into more than 3 nested lists it'll just repeat the last list-style, ie: lower-roman or square. I wasn't sure whether more nesting would be required or if it was overkill, thus the:
  ```
  @include nestedListStyles {
    @include nestedListStyles;
  }
  ```
  If we don't need it, we can probably get rid of the @content 😄

Reply:
Oh oops, sounds like I misunderstood how this was working

Reply:
That said having 6 potential nested lists seem very unlikely -  the `@include nestedListStyles` was just saved on writing really verbose scss - its really a judgment call on whether we want to support anything more the 3 levels deep in a list

Reply:
I've restored it :)